src/amo/components/Addon/index.js

```diff
@@ -308,6 +314,21 @@ export class AddonBase extends React.Component {
             <h1 className="Addon-title" {...titleProps} />
             <p className="Addon-summary" {...summaryProps} />
 
+            <div className="Addon-badges">
+              {isFeatured ? (
+                <Badge
+                  type="featured"
+                  label={i18n.sprintf(i18n.gettext('Featured %(addonType)s'), { addonType })}
```

Comment:
Sadly, `addonType` is not translated. We have to handle this in a few places by doing if/else or switch statements for all the different addon types... so it seems what we really need is an `i18n.addonType` helper.